pyaedt/generic/general_methods.py

```diff
@@ -376,10 +377,81 @@ def open_file(file_path, file_options="r"):
             return open(local_file, file_options)
     elif os.path.exists(dir_name):
         return open(file_path, file_options)
+    elif settings.remote_rpc_session and settings.remote_rpc_session.filemanager.pathexists(dir_name):
+        return settings.remote_rpc_session.open_file(file_path, file_options)
     else:
         settings.logger.error("The file or folder %s does not exist", dir_name)
 
 
+@pyaedt_function_handler()
+def read_configuration_file(file_path):
+    """Parse a file and return information in form of list or dict.
+
+    Parameters
+    ----------
+    file_path : str
+        Full path to the file. Supported formats are ``toml``, ``json``,``tab``,``csv``,``xlsx``.
```

Comment:
```suggestion
        Full path to the file. Supported formats are ``"csv"``, ``"json"``, ``"tab"``, ``"toml"``, and ``"xlsx"``.
```
Alphabetize list when providing options. Because they are strings, quotation marks are needed.